app/views/passenger_mailer/notify_archived.haml

```diff
@@ -0,0 +1,3 @@
+#{@passenger.name},
+
+Your account at Special Transit has been archived.
```

Comment:
You should set your text editor so it puts a newline at the EOF. Read [this](https://thoughtbot.com/blog/no-newline-at-end-of-file) for more information.